app/src/main/java/com/example/joshuarodriguez/mvpcalculator/mvp/mvp/view/CalculatorView.java

```diff
@@ -56,136 +56,134 @@
     @BindView(R.id.button_equals) Button mButtonEquals;
     @BindView(R.id.button_clear) Button mButtonClear;
```

Comment:
You should not use this format type with "m" at first position(ej: mButtonClear). Instead use buttonClear.